api/src/main/java/org/openmrs/module/mirebalaisreports/definitions/helper/DailyIndicatorByLocationReportDefinition.java

```diff
@@ -77,12 +75,12 @@ public ReportDefinition constructReportDefinition() {
      */
     public List<Location> getLocations() {
         List<String> skip = new ArrayList<String>();
-        skip.addAll(Arrays.asList(Locations.UNKNOWN.uuid(), MirebalaisLocations.MIREBALAIS_HOSPITAL.uuid(), MirebalaisLocations.MIREBALAIS_CDI_PARENT.uuid()));
+        skip.addAll(Arrays.asList("Unknown Location", "Hôpital Universitaire de Mirebalais - Prensipal", "Mirebalais"));
```

Comment:
Why are we looking up by name here?  Unlike all of the other methods below, this is not a test.  I'd recommend either looking up by uuids, or using this as the initial example of how we'd use constants.json in our config projects, and set json key values that we can then reference in the code.